docs/install/vm-install/rhel.md

```
@@ -106,9 +106,9 @@ When troubleshooting issues with Admin UI access, it's advisable to check the [l
 ```text
 https://FQDN/jans-casa
 ```
-## Enabling HTTPS 
-To enable communication with Janssen Server over TLS (https) in a production 
-environment, Janssen Server needs details about CA certificate. Update the 
+## Enabling HTTPS
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)